src/main/java/com/shapesecurity/bandolier/es2017/bundlers/PiercedModuleBundler.java

```diff
@@ -26,7 +26,6 @@
 	@Override
 	public @NotNull Script bundleEntrypoint(BundlerOptions options, String entry, Map<String, Module> modules) {
 		HashTable<String, Module> newModules = HashTable.emptyUsingEquality();
-		HashTable<String, ImmutableSet<Pair<String, String>>> removedImports = HashTable.emptyUsingIdentity();
 		for (Map.Entry<String, Module> mapEntry : modules.entrySet()) {
 			Module module = mapEntry.getValue();
 			module = new Module(module.directives, module.items.map(item -> {
```

Comment:
This isn't really relevant to this PR, but it looks to me like the `if (options.exportStrategy == BundlerOptions.ExportStrategy.ALL_GLOBALS)` below could be pulled out of the lambda, no? So that you'd only perform the `.map` at all if the test there is satisfied.